<!DOCTYPE HTML>
<!--
	Theory by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>
	<head>
		<title>Elements - Theory by TEMPLATED</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="subpage">

		<!-- Header -->
			<header id="header">
				<div class="inner">
					<a href="index.html" class="logo">MAJOR LAB</a>
					<nav id="nav">
						<a href="index.html">Home</a>
						<a href="research.html">Research</a>
						<a href="publications.html">Publications</a>
						<a href="news.html">News</a>
						<a href="equipment.html">Resources</a>
						<a href="positions.html">Positions</a>
						<a href="alumni.html">Alumni</a>
						<a href="contact.html">Contact</a>
					</nav>
					<a href="#navPanel" class="navPanelToggle"><span class="fa fa-bars"></span></a>
				</div>
			</header>

		<!-- Main -->
			<section id="main" class="wrapper">
				<div class="inner">
					<header class="align-center">
						<h2><b>Research Overview</b></h2>
					</header>

				<!-- Content -->
					<h4 align="justify">Although initiated by genetic mutation, the unchecked proliferation, aberrant differentiation, and altered motility of cancer cells depends upon the integrity and activation state of specific signal transduction pathways.
						Our laboratory is interested in understanding how alterations in these signaling pathways contribute to human cancer, and whether exploitation of that understanding can aid in the development of new diagnostics, prognostics and therapeutic intervention strategies.
						To this end, we employ a global “systems level” integrative discovery platform, one that has as a foundation mass spectrometry-based proteomic interaction networks. More specifically, through LC/MS/MS, we define the physical interaction network for a signaling pathway of oncogenic interest.
						By small molecule and functional genomic screening, we then annotate the human genome for functional contribution to the pathway of interest. Integration of these data with cancer-associated mutation data and cancer-associated gene expression data yields a powerful tool for oncogenic discovery—a cancer annotated physical/functional map for a specific signaling pathway of interest.
						The models and hypotheses produced though integrative screening are challenged through mechanistic studies employing cultured human cancer cells, zebrafish, mice and in vitro biochemical systems. With this general approach, we are currently pursuing the following projects:
					</h4>
					<div class="row">
						<div class="6u 12u$(small)">
							<h2 class=align-center"><b>KEAP1 and NRF2 Signaling in Cancer</b></h2>
							<h4 align="justify">KEAP1 is an E3 ubiquitin ligase important for cellular defense against oxidative stress, and in that context contributes fundamentally to aging, neurodegeneration, and a myriad of human cancers, most notably lung cancer. KEAP1 functions by ubiquitylating the NRF2 transcription factor, resulting in NRF2 degradation.
								In cancer, mutations within NRF2 or KEAP1 result in constitutive NRF2-driven expression of cytoprotective genes—this occurs in ~30% of lung cancer. In an effort to better understand KEAP1/NRF2, we were the first to define and validate a KEAP1 protein-protein interaction network.
								This revealed a group of KEAP1-associated proteins that competitively displaces NRF2, driving pathway activation.
								<br>
								<br>
								Our work has also begun to connect KEAP1 genotype with phenotype, wherein we functionally and biochemically characterized 19 lung cancer KEAP1 mutations.
								This revealed that ~half of the KEAP1 cancer mutations are hypomorphic, and surprisingly retain the ability to ubiquitylate NRF2. Ongoing work is focused on the mechanism by which these ‘ANCHOR’ mutants impair KEAP1-dependent degradation (but not ubiquitylation) of NRF2 function.
								Based off of proteomic, functional genomic and pharmacological screens, we are also pursuing novel NRF2-independent functions for KEAP1, including a cell cycle phenotype relevant for lung cancer cell proliferation.
								Most recently, we have created a new NRF2-active mouse model, allowing us to test the importance of our discoveries and the various biological impacts of NRF2 in a physiologically relevant system.
							</h4>
						</div>
						<div class="6u 12u$(small)">
							<h2 class=align-center"><b> Protein-protein Network Discoveries</b></h2>
							<h4 align="justify"> We are working within a larger team to illuminate the understudied kinome, a set of 162 kinases that remain relatively uncharacterized. My team is defining the protein-protein interaction and proximity networks for these kinases.
								Additionally, we are ‘binning’ the understudied kinases into signal transduction pathways using arrayed ORF screens and Crispr-based gain and loss-of-function screens. We have developed an extensive expertise in both affinity and immunoprecitipation purification assisted
								mass spectrometry as well as utilizing proximity based purification methods to define protein-protein interactions.
							</h4>
							<h2 class=align-center"><b>Phosphoproteomics and MIBs Profiling</b></h2>
							<h4>We have devoted great effort to kinase enrichment mass spectrometry to capture, identify and quantify the kinome in a single mass spectrometry run. In a recent example, we used this approach in a competitive fashion to define the kinase target landscape for FDA-approved kinase inhibitors.
							</h4>
						</div>
					</div>
					<div class="row">
						<div class="6u 12u$(small)">
							<h2 class=align-center"><b>Mechanistic Studies of WNT/β-catenin Signaling</b></h2>
							<h4 align="justify">Of the relatively small number of signaling pathways that function as master regulators of development, adult tissue homeostasis and cancer, the β-catenin dependent Wnt pathway (Wnt/β-catenin) figures prominently; it regulates the growth and fate of neoplastic cells in tissues of diverse origin, notably the colon, kidney, breast and skin.
								My group has performed an array of proteomic and functional genomic studies of WNT signaling, including protein-protein interaction screens, kinase enrichment profiling, phospho-proteomics, siRNA and haploid mutagenesis loss-of-function screens, and more recently, novel gain-of-function screens.
								Integration of these data has and continues to reveal mechanistic insight and new disease-relevant regulators of pathway activity. As an example, our gain-of-function genomic screens demonstrated that the FOXP1 transcription factor activates β-catenin dependent transcription.
								Proteomic analyses demonstrated that FOXP1 binds the β-catenin transcriptional complex on chromatin. Disease-focused studies in mice and human clinical samples demonstrated that FOXP1 overexpression in B-cell lymphoma activates WNT signaling to promote tumor growth.
								In more recent work, we discovered that the ubiquitin-specific protease (USP6) deubiquitylates the WNT receptor to govern its endocytosis. We also discovered a WNT-driven negative feedback loop that activates the AAK1 kinase to promote endocytosis of LRP6.
							</h4>
						</div>
					</div>
				</div>
				<hr class="major" />

				<!-- sponsors-->
				<section class="wrapper">
					<div class="inner">
						<h2 align="center"><b> The Major lab is grateful for all of the support we have received over the years in our efforts to better understand cancer.</b> </h2>
						<div class="image fit">
							<a href="https://druggablegenome.net/" target="_blank">  <img src="images/logos/IDG2.jpg" alt="IDG" /></a>
						</div>
						<div class="box alt">
							<div class="row 50% uniform">
								<div class="4u"><a href="http://cancer.org"> <span class="image fit"> <img src="images/logos/spic1.jpg" alt="ACS" /> </span> </a> </div>
								<div class="4u"><a href=""> <span class="image fit"><img src="images/logos/spic2.jpg" alt="NIH" /> </span> </a> </div>
								<div class="4u$"><a href=""><span class="image fit"><img src="images/logos/spic7.jpg" alt="Angel" /> </span> </a> </div>
								<!-- Break -->
								<div class="4u"><a href=""><span class="image fit"><img src="images/logos/spic4.jpg" alt="Golfers" /></span> </a> </div>
								<div class="4u"><a href=""><span class="image fit"><img src="images/logos/spic5.jpg" alt="SFA" /></span> </a> </div>
								<div class="4u$"><a href=""><span class="image fit"><img src="images/logos/spic6.jpg" alt="V" /></span> </a> </div>
							</div>
						</div>
					</div>
			</section>

				<div class="row 200%">
					<div>
						<h3 align="center"><b>Collaborators</b></h3>
						<ul>
							<li> <a href="https://unclineberger.org/directory/bernard-weissman/">Weissman lab</a> </li>
							<li> <a href="https://sites.google.com/site/thechenlabatnccu/">Chen lab </a></li>
							<li> <a href="https://www.med.unc.edu/marsicolunginstitute/directory/scott-h-randell-phd-2/">Randell lab</a></li>
							<li> <a href="https://kutluaylab.wustl.edu/"> Kutluay lab </a></li>
							<li> <a href="https://sites.wustl.edu/djuranoviclab/">Djuranovic lab </a></li>
							<li> <a href="https://unclineberger.org/pecotlab/">Pecot lab</a> </li>
						</ul>
					</div>
				</div>

		<!-- Footer -->
		<footer id="footer">
			<div class="inner">
				<div class="flex flex-2">
					<div class="copyright">
						&copy; Washington University in St. Louis.
						<br>Design: <a href="https://templated.co">TEMPLATED</a>.
						<br> <b>Updated: April 24, 2021.</b> .
					</div>
					<div class="image right">
						<a href="https://medicine.wustl.edu/" target="_blank"> <img src="images/logos/wustl.jpg" alt="WUSTL" /></a>
					</div>
				</div>
			</div>
		</footer>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>